### PR TITLE
Meet Orion

### PR DESCRIPTION
**Pull request by shashank-boyapally**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Orion is cli tool which automates the process of writing scripts when you need csv files to find regressions in metrics. 

To run orion on your local, please follow the following instructions.

### Build Orion
Building Orion is a straightforward process. Follow these commands:

Clone the current repository using git clone.

```
>> git clone <repository_url>
>> pip install venv
>> source venv/bin/activate
>> pip install -r requirements.txt
>> export ES_SERVER = <es_server_url>
>> pip install .
```
### Run Orion
Executing Orion is as simple as building it. After following the build steps, run the following:
```
>> orion
```

## Related Tickets & Documents

- Related Issue #1 
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: orion.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    Args:
+        config (str): path to the config file
+        debug (bool): lets you log debug mode
+        output (str): path to the output csv file
+    """
+    level= logging.DEBUG if debug else logging.INFO
+    logger = logging.getLogger("Orion")
+    logger= set_logging(level,logger)
+
```
Comment (jtaleric) [inline orion.py:36]: Lets add a check for the ES_SERVER

**Review by jtaleric — COMMENTED** [review]
one nit... we should check to see if the env is set, and that we can query the ES server.

File: orion.py (hunk 1 above)
Comment (shashank-boyapally) [reply orion.py:36]: sure, I will check of ES_SERVER

File: orion.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        config (str): path to the config file
+        debug (bool): lets you log debug mode
+        output (str): path to the output csv file
+    """
+    level = logging.DEBUG if debug else logging.INFO
+    logger = logging.getLogger("Orion")
+    logger = set_logging(level, logger)
+
+    if "ES_SERVER" not in os.environ:
+        logger.error("ES_SERVER not set")
```
Comment (jtaleric) [inline orion.py:41]: ES_SERVER environmental variable not set.

Comment (shashank-boyapally) [reply orion.py]: changed the error message.

**Review by jtaleric — APPROVED** [review]
lgtm!

**Pull request merged**